recipes/sambamba/meta.yaml

```diff
@@ -1,23 +1,40 @@
-{% set version='0.6.6' %}
+{% set version='0.6.7' %}
+{% set revision='223fa20' %}
+
 package:
   name: sambamba
   version: {{ version }}
 
+build:
+  number: 0
+
+extra:
+  skip-lints:
+    - uses_git_url
+
 source:
-  fn: sambamba_v{{ version }}.tar.bz2
-  url: https://github.com/lomereiter/sambamba/releases/download/v{{ version }}/sambamba_v{{ version }}_linux.tar.bz2 # [linux]
-  md5: 6fc3b70ac32441f3a9c6aa87743c8e3c # [linux]
-  url: https://github.com/lomereiter/sambamba/releases/download/v{{ version }}/sambamba_v{{ version }}_osx.tar.bz2 # [osx]
-  md5: 07d82a2102126565e6796658face7c71 # [osx]
+  git_url: https://github.com/biod/sambamba
+  git_rev: {{ revision }}
+  md5: unused
 
-build:
-  number: 1
+requirements:
+  build:
+    - ldc
+    - gcc
```

Comment:
Did you try to use conda build 3?
Like, replace gcc for:
```
- {{ compiler('c') }}
```

Reply:
I tried/changed the Makefile to accommodate GDC, but quite a few changes were required and Pjotr mentioned that LDC had more optimizations anyway (see comments above).